virtualization/Docker/Dockerfile.dev

```diff
@@ -1,64 +0,0 @@
-# Dockerfile for development
```

Comment:
Let's keep a dev dockerfile. Not everyone using VSCode with dev containers yet :)